### DISCUSSION TITLE
自己训练了文本检测模型，想要在Android上部署使用，缺乏资料

### DISCUSSION THREAD

**Discussion by lilinebo** [PaddlePaddle/PaddleOCR#14002] [Q&A]
请问在https://github.com/PaddlePaddle/PaddleOCR/tree/main/deploy/fastdeploy/android 中提到的“用 Android Studio 打开 PP-OCRv3/android 工程”我没找到在哪里。现在的Androiddemo支持v3吗。我看的不是很明白

**Comment by GreatV** [comment]
https://github.com/PaddlePaddle/PaddleOCR/tree/main/deploy/fastdeploy/android 这个目录下不是android工程吗

**Comment by Liyulingyue** [comment]
建议参考此文档 https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/pipeline_deploy/edge_deploy.md
